capstone/src/main/webapp/rider.js

```diff
@@ -0,0 +1,12 @@
+//used for testing purposes only
+function getRider(){
+  fetch('/rider?alt=json').then(response => {
```

Comment:
It looks like rider gives back json unconditionally so probably don't need the alt=json part (unless you plan to add a default HTML mode to /rider at some point).